src/views/PeopleSearch/components/PeopleSearchResult/index.tsx

```diff
@@ -81,42 +88,72 @@ const PeopleSearchResult = ({ person, lazyLoadAvatar }: Props) => {
       break;
   }
 
+  const emailIcon = isMobileView ? (
+    <></>
+  ) : (
+    <div className={styles.mailingIconContainer}>
+      <CardActionArea className={styles.mailCardAction}>
+        <a href={`mailto:${person.Email}`}>
+          <MailOutlineIcon
+            sx={{
+              color: gordonColors.neutral.grayShades[50],
+              height: '100%',
+              width: 40,
+              borderRadius: 2,
+            }}
+          />
+        </a>
+      </CardActionArea>
+    </div>
+  );
```

Comment:
You can actually set this to null if it is mobile view. A nice shortcut for specifying "only show this thing if this other thing is true" is:
```suggestion
  const emailIcon = !isMobileView && (
    <div className={styles.mailingIconContainer}>
      <CardActionArea className={styles.mailCardAction}>
        <a href={`mailto:${person.Email}`}>
          <MailOutlineIcon
            sx={{
              color: gordonColors.neutral.grayShades[50],
              height: '100%',
              width: 40,
              borderRadius: 2,
            }}
          />
        </a>
      </CardActionArea>
    </div>
  );
```